src/vue/components/nav/overlay-nav.vue

```diff
@@ -105,7 +105,7 @@
       </alph-tooltip>
 
       <alph-tooltip :tooltipText="l10n.getText('TOOLTIP_WORD_USAGE')" tooltipDirection="left"
-                  v-show="$store.state.app.wordUsageExamplesReady">
+                  v-show="$store.state.app.homonymDataReady">
```

Comment:
I think we have to have a language capability for this, like we do with the grammars, because for now only Latin has this functionality.

Reply:
e.g. LanguageModelFactory.getLanguageModel(homonym.languageID).grammarFeatures()

Reply:
I have added additional state property - `showWordUsageTab` and it checks `enableWordUsageExamples` for defining if to show wordUsageExamples tab.